examples/reagent/src/cljs/todomvc/actions.cljs

```diff
@@ -0,0 +1,39 @@
+(ns todomvc.actions
+  (:require [todomvc.session :as session]
```

Comment:
Ah, misunderstood, thought you were just referring to `project.clj`. I will go back and tab indent all of the files in `src/`